doc-pages/src/static/docs/upgrade-guide.md

```diff
@@ -1,116 +0,0 @@
-# Upgrade Guide
-
-## Upgrading from deck.gl v3 to v4
```

Comment:
I would argue that we do want an upgrade guide so I'd leave a place holder here and in the index.